app/grandchallenge/evaluation/templates/evaluation/evaluation_detail.html

```diff
@@ -205,6 +205,19 @@ <h3 class="card-title">Contact User</h3>
 
             {% include "evaluation/evaluation_incomplete_jobs_detail.html" %}
 
+            <div class="card-body">
+                <h3 class="card-title">Error Message</h3>
```

Comment:
Since the message in this section is not necessarily an error (e.g. in the case of incomplete jobs), should the title be renamed to "Status Message" rather than "Error Message"?